tests/Serializer/Type/ParserTest.php

```diff
@@ -72,14 +92,6 @@ public function validTypesProvider(): iterable
             'Foo<"asdf asdf">',
             $type('Foo', ['asdf asdf']),
         ];
-        yield [
```

Comment:
removed this as it will not allow `''` (empty strings syntax) and was not available in the 1.x parser

Reply:
From what I can guess, what you were trying to do here was handled by YAML